changelog/963.feature.rst

```diff
@@ -0,0 +1 @@
+Further developed grid objects and improved their documentation, including several example notebooks.
```

Comment:
```suggestion
Further developed grid objects (`plasmapy.plasma.grids`) and improved their
documentation, including several example notebooks.
```

The `plasmapy.plasma.grids` will provide a link back to the documentation of the module when the change log is compiled.

I'd also like to see a few specifics on how the objects where "further developed."  What was actually done?